Project.toml

```diff
@@ -17,6 +17,7 @@ GaussQuadrature = "d54b0c1a-921d-58e0-8e36-89d8069c0969"
 GilbertCurves = "88fa7841-ef32-4516-bb70-c6ec135699d9"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
+JET = "c3a54625-cd67-489e-a8e7-0a5a0ff4e31b"
```

Comment:
We should add this to the perf environment, I thought I had, but I guess that was in a different repo